src/main/java/org/osc/controller/nuage/api/NuageSecurityControllerApi.java

```diff
@@ -249,6 +252,41 @@ public Element registerRedirectionTarget(InspectionPortElement inspectionPort, S
         List<Domain> dms = fetcher.fetch(filter, null, null, null, null, null, Boolean.FALSE);
         if (!CollectionUtils.isEmpty(dms)) {
             selectDomain = dms.get(0);
+
+            List<NetworkElement> inspectionPorts = new ArrayList<>();
+            inspectionPorts.add(inspectionPort.getIngressPort());
+            if (!inspectionPort.getEgressPort().getElementId().equals(inspectionPort.getIngressPort().getElementId())) {
+                inspectionPorts.add(inspectionPort.getEgressPort());
+            }
+
+            // Create PolicyGroup with Ingress and Egress inspection ports
+            NetworkElement pg = createPolicyGroup(inspectionPorts, domainId);
+
+            // Create a Forwarding Policy with priority 0
+            Optional<IngressAdvFwdTemplate> BypassFwdPolicyTemp = getForwardingPolicies(selectDomain).stream().filter(x -> x.getName().equals(BYPASS_FWDPOLICY_NAME)).findFirst();
+            IngressAdvFwdTemplate BypassFwdPolicy = BypassFwdPolicyTemp.orElse(null);
+
+            if (BypassFwdPolicy == null) {
+                BypassFwdPolicy = new IngressAdvFwdTemplate();
+                BypassFwdPolicy.setActive(true);
+                BypassFwdPolicy.setPriority(Long.valueOf(1));
```

Comment:
recommend adding the 1 as a const value with a meaningful name as well given that it is an important value for this approach.